### ISSUE TITLE
[macOS bundle] locale & $PATH issues

### ISSUE THREAD

**Issue by ahmedelgabri** [jwilm/alacritty#645]
When I launch alacritty using the [MacOS application bundle](https://github.com/jwilm/alacritty/wiki/MacOS-application-bundle) I have two problems:

### locale issue:

<img width="437" alt="screen shot 2017-07-01 at 20 17 13" src="https://user-images.githubusercontent.com/63876/27764481-bd535ea4-5e9a-11e7-883c-4969366cd449.png">

I have [`$LANG` & `$LC_ALL ` set to `en_US.UTF-8`](https://github.com/ahmedelgabri/dotfiles/blob/master/zsh/.zprofile#L42-L45)

But `alacritty` is querying the [OS instead](https://github.com/jwilm/alacritty/blob/master/assets/osx/Alacritty.app/Contents/MacOS/launcher#L7)

---

### `$PATH` issue:

For some reason the bundle also misses a couple of entries from my [`$PATH`](https://github.com/ahmedelgabri/dotfiles/blob/master/zsh/.zprofile#L73-L81)

Using the app bundle:

<img width="345" alt="screen shot 2017-07-01 at 20 32 18" src="https://user-images.githubusercontent.com/63876/27764559-9fae0190-5e9c-11e7-8ade-b8543c2a32d0.png">

Using the excutable directly from `target/release/alacritty`:

<img width="340" alt="screen shot 2017-07-01 at 20 32 44" src="https://user-images.githubusercontent.com/63876/27764561-b0370b42-5e9c-11e7-8e9a-6ed7a912af7f.png">

There are no issues when I use the executable directly `target/release/alacritty`, they only happened when I use the macOS bundled app.

**Comment by z3t0** [comment]
I am having a similar issue where the PATH is picked up from the OS.

**Comment by rafi** [comment]
Have you tried setting login shell? e.g.
```
shell:
  args:
    - --login
```

**Comment by ahmedelgabri** [comment]
@rafi no, because I assumed that if it works with executable it will work with the app.

But I tried it & it fixed the `$PATH` issue but I had to pass also `program`, `args` alone was not enough.

```yaml
shell:
 program: /usr/local/bin/zsh
 args:
   - --login
```

**Comment by z3t0** [comment]
@ahmedelgabri thanks that worked for me.

**Comment by sodiumjoe** [comment]
sounds like the `$PATH` issue is fixed with `--login`, and I believe #1087 will fix the `locale` issue. Do you mind trying that branch?
